instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/resources/OtelSpringResourceProperties.java

```diff
@@ -9,6 +9,7 @@
 import java.util.Map;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+@Deprecated // Use otel.resource.attributes instead
 @ConfigurationProperties(prefix = "otel.springboot.resource")
```

Comment:
sorry, I'm still a bit confused because there's still lots of stuff in this PR about resources, but the PR title only mentions `otel.propagators`? (the title is what we use to auto-generate the change log, so if there's additional stuff then we will miss mentioning it in the release notes)

is `otel.propagators` change independent from the resources changes? if so, can you split them into separate PRs? thx

Reply:
great idea - I'll do that.

For resource attributes, it's https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/10511